kaldb/src/main/java/com/slack/kaldb/chunk/ChunkInfo.java

```diff
@@ -135,34 +187,27 @@ public boolean containsDataInTimeRange(long startTimeMs, long endTimeMs) {
    * Update the max and min data time range of the chunk given a new timestamp.
    */
   public void updateDataTimeRange(long messageTimeStampMs) {
-    if (dataStartTimeEpochMs == 0 || dataEndTimeEpochMs == 0) {
-      dataStartTimeEpochMs = messageTimeStampMs;
-      dataEndTimeEpochMs = messageTimeStampMs;
+    if (dataEndTimeEpochMs == MAX_FUTURE_TIME) {
+      dataStartTimeEpochMs = Math.min(dataStartTimeEpochMs, messageTimeStampMs);
+      dataEndTimeEpochMs = Math.min(dataEndTimeEpochMs, messageTimeStampMs);
```

Comment:
I'm not sure this makes sense - since we already checked for equality of `dataEndTimeEpochMs` to that of `MAX_FUTURE_TIME` this is in effect:

```
dataEndTimeEpochMs = Math.min(MAX_FUTURE_TIME, messageTimeStampMs);
```
When would we expect `messageTimeStampMs` to be larger than that of `MAX_FUTURE_TIME`? Shouldn't this just be:
```
dataEndTimeEpochMs = messageTimeStampMs;
```